RxExample/RxExample/Examples/GitHubSignup/DefaultImplementations.swift

```diff
@@ -105,8 +105,13 @@ class GitHubDefaultAPI : GitHubAPI {
     func signup(_ username: String, password: String) -> Observable<Bool> {
         // this is also just a mock
         let signupResult = arc4random() % 5 == 0 ? false : true
-        return Observable.just(signupResult)
-            .concat(Observable.never())
+        
```

Comment:
Hi @gorozco58 ,

I think we should just do `delay(0.4, scheduler: MainScheduler.instance)` instead of `throttle(0.4, scheduler: MainScheduler.instance)`.